easybuild/easyconfigs/l/libfabric/libfabric-1.12.1-GCCcore-10.3.0.eb

```diff
@@ -56,6 +58,9 @@ configopts = "--disable-sockets "
 # osdependencies.append(('libnl3-devel', 'libnl3-dev'))
 configopts += "--disable-usnic "
 
+# Disable PSM3 as this fails to build on machine without AVX
+configopts += "--disable-psm3 "
```

Comment:
Is this still necessary? We aren't doing this for version 1.13.0. I thought the other patch took care of this?

Reply:
It was necessary for me yes. I'm compiling with `optarch = march=core2` on an Intel E5-2660. We are doing this because some of our nodes are older (and some are newer.. )

Reply:
Seems a bit drastic to me to do this because it doesn't work on system that don't have AVX (which are pretty old...).

@ysagon This seems like a site-specific problem to me, maybe it's better to fix this with an EasyBuild hook (see https://docs.easybuild.io/en/latest/Hooks.html)?

Reply:
Another, perhaps better way to do this could be to implement a custom easyblock for `libfabric` that automatically uses `--disable-psm3` if the build host does not support `AVX`, or if `--optarch=generic` is used.